clients/spark/core/src/main/scala/io/treeverse/clients/BulkRemoverFactory.scala

```diff
@@ -19,7 +13,7 @@ import org.apache.hadoop.conf.Configuration
 import java.net.URI
 import java.nio.charset.Charset
 import java.util.stream.Collectors
-import collection.JavaConverters._
+import scala.collection.JavaConverters._
```

Comment:
Why this change?

Reply:
❓

Reply:
It was an automatic import optimization, but there's no difference- both imports point to the same library